src/components/Button.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Button({ name }) {
+  return <button type="button">{name}</button>;
+}
+
+export default Button;
```

Comment:
- enforce `name` to be string with `propTypes` [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]